src/main/java/frc/robot/Constants.java

```diff
@@ -11,7 +11,7 @@
 
     public static class Intake {
         public static final double DEFAULT_POWER = 0.5; // power intake will receive on the basic command. [%]
-        public static final double SLOPE = -3./16; // 0.5 is a temp number (still hasnt been tested). [% / m/s]
+        public static final double POWER_TO_VELOCITY = -3 / 16.0; // Ratio of power to velocity. [% / m/s]
```

Comment:
I don't want to nitpick but the word "ratio" is pretty important here IMO. The name currently implies some sort of mapping, not a ratio.